.travis.yml

```diff
@@ -46,9 +46,10 @@ before_script:
   # Change Galaxy configuration to use the newly created PostgreSQL database
   - sed -e 's|^#database_connection.*|database_connection = postgresql://postgres:@localhost/galaxy-travis|' $GALAXY_CONFIG_FILE.sample > $GALAXY_CONFIG_FILE
   # Setup Galaxy master API key and admin user
-  - GALAXY_MASTER_API_KEY=$(cat /dev/urandom | LC_ALL=C tr -dc A-Za-z0-9 | head -c 32)
-  - GALAXY_USER_EMAIL=${USER}@localhost.localdomain
-  - sed -i.bak -e "s/^#master_api_key.*/master_api_key = $GALAXY_MASTER_API_KEY/" -e "s/^#admin_users.*/admin_users = $GALAXY_USER_EMAIL/" $GALAXY_CONFIG_FILE
+  - export BIOBLEND_GALAXY_MASTER_API_KEY=$(cat /dev/urandom | LC_ALL=C tr -dc A-Za-z0-9 | head -c 32)
+  - export BIOBLEND_GALAXY_USER=${USER}
+  - GALAXY_USER_EMAIL=${BIOBLEND_GALAXY_USER}@localhost.localdomain
```

Comment:
I think it would be better to export `GALAXY_USER_EMAIL` and drop `BIOBLEND_GALAXY_USER`, see below.